BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -55,6 +65,17 @@ public BraintreeClient(@NonNull Context context, @NonNull String authorization)
         this(createDefaultParams(context, authorization));
     }
 
+    /**
+     * Create a new instance of {@link BraintreeClient} using a tokenization key or client token.
```

Comment:
Let's add a note in the description that this constructor should only be used for applications with multiple activities that need to handle deep link returns (or some wording like that)

Reply:
Yeah that's a good idea I'm down to add that in.